apps/web/controllers/pets/edit.rb

```diff
@@ -7,6 +7,7 @@ class Edit
     expose :pet
 
     def call(params)
+      # TODO How I can apply more security here?
       @pet = PetRepository.find(params[:id])
```

Comment:
```ruby
# this code don't work :P
# the `find` is ignored
@pet = PetRepository.all_owned_by(current_user).find(params[:id])
```

I think in improve using something like `PetRepository.some_query(current, id)` :)